src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -233,7 +233,7 @@ public function checkAnnotations(\ReflectionClass $refl, $class)
         // Detect annotations on the class
         if (false !== $doc = $refl->getDocComment()) {
             foreach (['final', 'deprecated', 'internal'] as $annotation) {
-                if (false !== \strpos($doc, $annotation) && preg_match('#\n\s+\* @'.$annotation.'(?:( .+?)\.?)?\r?\n\s+\*(?: @|/$)#s', $doc, $notice)) {
+                if (false !== \strpos($doc, $annotation) && preg_match('#\n\s+\* @'.$annotation.'(?:( .+?)\.?)?\r?\n\s+\*(?: @|/$|\r?\n)#s', $doc, $notice)) {
```

Comment:
detects blank lines after `@final`/`@deprecated`/`@internal` annotations (detected by the fixtures update made in df1d50dcc2eb9ea1fce54ea9eb557247ca5b349a)